core/src/http/AuthHttpClient.js

```diff
@@ -576,6 +576,43 @@ export class AuthHttpClient {
         };
     }
 
+    /**
+     * get information about a particular standing order
+     *
+     * @param {*} accountId
+     * @param {*} standingOrderId
+     * @param {*} keyLevel
+     * @returns
+     * @memberof AuthHttpClient
+     */
+    async getStandingOrder(accountId, standingOrderId, keyLevel) {
```

Comment:
move these two before `tokenOperationSignature` because that one isn't an API call.